README.md

```diff
@@ -23,3 +23,40 @@ gradle bootRun --refresh-dependencies
+| IEXEC_CORE_HOST                               | Host to connect to the Scheduler.                                                                                                | String           | localhost                                                 |
+| IEXEC_CORE_PORT                               | Port used by the Scheduler.                                                                                                      | Positive integer | 13000                                                     |
+| IEXEC_WORKER_NAME                             | Name the worker should be known by.                                                                                              | String           | worker                                                    |
+| IEXEC_WORKER_BASE_DIR                         | Path to the folder within which the worker will read and write tasks' inputs and outputs.                                        | String           | /tmp/iexec-worker                                         |
+| IEXEC_WORKER_OVERRIDE_AVAILABLE_CPU_COUNT     | Number of CPUs to use. If not set, n-1 CPUs will be used, where n is the number of available processors.                         | Positive integer |                                                           |
+| IEXEC_WORKER_GPU_ENABLED                      | Whether to use the GPU instead of the CPU. Note that if it is true, `IEXEC_WORKER_OVERRIDE_AVAILABLE_CPU_COUNT` will be ignored. | Boolean          | false                                                     |
+| IEXEC_GAS_PRICE_MULTIPLIER                    | Transactions will be sent with `networkGasPrice * IEXEC_GAS_PRICE_MULTIPLIER`.                                                   | Float            | 1.3                                                       |
+| IEXEC_GAS_PRICE_CAP                           | In Wei, will be used for transactions if `networkGasPrice * IEXEC_GAS_PRICE_MULTIPLIER > gasPriceCap`.                           | Integer          | 22000000000                                               |
+| IEXEC_WORKER_OVERRIDE_BLOCKCHAIN_NODE_ADDRESS | If set, will be used instead of the address given by the blockchain adapter.                                                     | String           |                                                           |
+| IEXEC_DEVELOPER_LOGGER_ENABLED                | Whether to print apps' logs.                                                                                                     | Boolean          | false                                                     |
+| IEXEC_WORKER_TEE_COMPUTE_MAX_HEAP_SIZE_GB     | Max heap size for TEE apps.                                                                                                      | Positive integer | 8                                                         |
```

Comment:
Do we have minimum requirement for the worker ? (CPU / RAM)
What happens if the provided size is bigger than system available memory ?

Reply:
I don't actually know. That'll be points we will have to work on.

Reply:
Current implementation is not optimal but protects workers.
In the future, a worker should expose to the core how much RAM it has hence the Scheduler would be able to schedule tasks on compatible workers.